src/server/routes/actions.ts

```diff
@@ -53,61 +37,104 @@ const addressMiddleware = (
 
 type PartialDrip<T extends FaucetRequestType | BotRequestType> = Partial<T> & Pick<T, "address">;
 
-const dripRequestHandler = new DripRequestHandler(actions, storage, recaptchaService);
+export const createActionsRouter = (
+  opts: { type: "bot" } | { type: "web"; webConfig: WebBackendConfig },
+  actions: Actions,
+) => {
```

Comment:
Correct me if I'm wrong.

This checks if it's either a bot or a web deployment, and based on that it only enables ONE of the endpoints. Right?

Reply:
Yes, that was it.